### ISSUE TITLE
Color options

### ISSUE THREAD

**Issue by PranjalDesai** [drozdzynski/Steppers#12]
Is there a way to change the color of the buttons. It shows up as neon green for me, I would like it to be the accent color of the app.

Thanks

**Comment by drozdzynski** [comment]
Buttons by default get color from accent color.

**Comment by PranjalDesai** [comment]
Not for me. My accent color is a pink color but instead i'm getting a teal color. am I missing anything?

**Comment by drozdzynski** [comment]
Did you use appcompat theme?

**Comment by PranjalDesai** [comment]
Yes that's what I used

**Comment by drozdzynski** [comment]
I use it in layout with a button "@style/Widget.AppCompat.Button.Colored" so it should get color from your app theme, but if it not works, I can try to add option "setColor" which will change circle step color and button color or setPositiveButtonColor which will change only button color. Is it good solution for you?

**Comment by PranjalDesai** [comment]
yeah that would be great!!![bitmoji](https://render.bitstrips.com/v2/cpanel/9742934-30933333_8-s1-v1.png?transparent=1&width=246)

**Comment by Abhi-Kathiriya** [comment]
Can anyone provide a source code of how to change button and circle color. I don't know how to change color
